src/utils/slpFile.ts

```diff
@@ -221,7 +247,33 @@ export class SlpFile extends Writable {
         ]);
       });
 
-      // Close characters and player
+      // Close characters
+      footer = Buffer.concat([footer, Buffer.from("}")]);
+
+      // Start names key for this player
+      footer = Buffer.concat([footer, Buffer.from("U"), Buffer.from([5]), Buffer.from("names{")]);
+
+      // Write display name
+      footer = Buffer.concat([
+        footer,
+        Buffer.from("U"),
+        Buffer.from([7]),
+        Buffer.from("netplaySU"),
+        Buffer.from([player.names.netplay.length]),
+        Buffer.from(`${player.names.netplay}`),
```

Comment:
Can you make the top level: `netplay` with the attributes `netplay.name` and `netplay.code`?